unity-renderer/Assets/Rendering/ProceduralSkybox/ToolProceduralSkybox/Scripts/Editor/Procedural Skybox/SkyboxEditor.cs

```diff
@@ -0,0 +1,1344 @@
+        {
+            for (int i = 0; i < selectedConfiguration.slots.Count; i++)
+            {
+                selectedConfiguration.slots[i].UpdateSlotsID(i);
+            }
+        }
+
+        private void RenderTimePanel()
+        {
+
+            GUILayout.Label("Preview", EditorStyles.boldLabel);
+
+            GUILayout.BeginHorizontal();
+            GUILayout.Label("Time : " + timeOfTheDay, EditorStyles.label, GUILayout.MinWidth(100));
+            timeOfTheDay = EditorGUILayout.Slider(timeOfTheDay, 0.01f, 24.00f, GUILayout.MinWidth(100), GUILayout.MaxWidth(300));
+            EditorGUILayout.LabelField((GetNormalizedDayTime() * 100).ToString("f2"), GUILayout.MaxWidth(50));
+
+            if (isPaused)
+            {
+                if (GUILayout.Button("Play"))
+                {
+                    ResumeTime();
+                }
+            }
+            else
+            {
+                if (GUILayout.Button("Pause"))
+                {
+                    PauseTime();
+                }
+            }
+
+            GUILayout.EndHorizontal();
+        }
+
+        void ResumeTime()
+        {
+            isPaused = false;
+            if (Application.isPlaying)
+            {
+                if (SkyboxController.i != null)
+                {
+                    SkyboxController.i.ResumeTime(true, timeOfTheDay);
+                }
+            }
+        }
+
+        void PauseTime()
+        {
+            isPaused = true;
+            if (Application.isPlaying)
+            {
+                if (SkyboxController.i != null)
+                {
+                    SkyboxController.i.PauseTime();
+                }
+            }
+        }
+
+        void RenderBackgroundColorLayer()
+        {
+
+            //GUILayout.Label("Background Layer", EditorStyles.boldLabel);
+            RenderColorGradientField(selectedConfiguration.skyColor, "Sky Color", 0, 24);
+            RenderColorGradientField(selectedConfiguration.horizonColor, "Horizon Color", 0, 24);
+            RenderColorGradientField(selectedConfiguration.groundColor, "Ground Color", 0, 24);
+            RenderHorizonLayer();
+        }
+
+        void RenderHorizonLayer()
+        {
+            EditorGUILayout.Separator();
+            RenderTransitioningFloat(selectedConfiguration.horizonHeight, "Horizon Height", "%", "value", true, -1, 1);
+
+            EditorGUILayout.Space(10);
+            RenderTransitioningFloat(selectedConfiguration.horizonWidth, "Horizon Width", "%", "value", true, -1, 1);
+
+            EditorGUILayout.Separator();
+
+            // Horizon Mask
+            RenderTexture("Texture", ref selectedConfiguration.horizonMask);
+
+            // Horizon mask values
+            RenderVector3Field("Horizon Mask Values", ref selectedConfiguration.horizonMaskValues);
+        }
+
+        void RenderAmbientLayer()
+        {
+
+            //GUILayout.Label("Ambient Layer", EditorStyles.boldLabel);
```

Comment:
```suggestion
```

Reply:
Done. Making all these changes in a new commit